### DISCUSSION TITLE
water spray in fds

### DISCUSSION THREAD

**Discussion by joker121-move** [firemodels/fds#13272] [Q&A]
Hello experts and teachers, I'd like to ask if in FDS (Fire Dynamics Simulator), does an increase in water spray pressure result in a change in the droplet size of the water mist produced?

**Comment by drjfloyd** [comment]
What does the FDS User's Guide say about the impact of pipe pressure?

**Reply by joker121-move** [reply]
Thank you for paying attention to this issue. I'd like to do some simulation studies by changing the droplet size of water. From my literature review, I found that the droplet size changes with increasing spray pressure—would this also be the case in FDS (Fire Dynamics Simulator)?  
If I set the droplet size here, will an increase in spray pressure have an effect? I couldn't find information on this in the FDS user manual.
&PART ID='my droplets', DIAMETER=1000., SPEC_ID='WATER VAPOR' /

**Reply by drjfloyd** [reply]
18.3.1 Sprinklers

Varying Pipe Pressure

**Comment by joker121-move** [comment]
thank you very much

---- Replied Message ----
| From | Jason ***@***.***> |
| Date | 08/04/2024 22:03 |
| To | firemodels/fds ***@***.***> |
| Cc | joker121-move ***@***.***>,
Author ***@***.***> |
| Subject | Re: [firemodels/fds] water spray in fds (Discussion #13272) |

18.3.1 Sprinklers

Varying Pipe Pressure
